example/poisson/poisson_package.cpp

```diff
@@ -86,22 +195,33 @@ TaskStatus UpdatePhi(T *u, T *du) {
     PARTHENON_REQUIRE_THROWS(dx == dy,
                              "UpdatePhi requires that DX be equal in all directions.");
   }
+  const Real dV = std::pow(dx, ndim);
 
-  parthenon::par_for(
-      DEFAULT_LOOP_PATTERN, "UpdatePhi", DevExecSpace(), 0, v.GetDim(5) - 1, kb.s, kb.e,
-      jb.s, jb.e, ib.s, ib.e,
-      KOKKOS_LAMBDA(const int b, const int k, const int j, const int i) {
-        dv(b, idphi, k, j, i) = -v(b, irho, k, j, i) * std::pow(dx, ndim);
-        dv(b, idphi, k, j, i) += v(b, iphi, k, j, i - 1) + v(b, iphi, k, j, i + 1);
-        if (ndim > 1) {
-          dv(b, idphi, k, j, i) += v(b, iphi, k, j - 1, i) + v(b, iphi, k, j + 1, i);
-          if (ndim == 3) {
-            dv(b, idphi, k, j, i) += v(b, iphi, k - 1, j, i) + v(b, iphi, k - 1, j, i);
-          }
-        }
-        dv(b, idphi, k, j, i) /= 2.0 * ndim;
-        dv(b, idphi, k, j, i) -= v(b, iphi, k, j, i);
-      });
+  StateDescriptor *pkg = pm->packages.Get("poisson_package").get();
+  if (isp_hi < 0) { // there is no sparse matrix, so we must be using the stencil
+    const auto &stencil = pkg->Param<Stencil_t>("stencil");
+    parthenon::par_for(
+        DEFAULT_LOOP_PATTERN, "StencilJacobi", DevExecSpace(), 0, v.GetDim(5) - 1, kb.s,
+        kb.e, jb.s, jb.e, ib.s, ib.e,
+        KOKKOS_LAMBDA(const int b, const int k, const int j, const int i) {
+          const Real rhs = dV * v(b, irho, k, j, i);
```

Comment:
Was the intention for this to be `v(b, irhs, k, j, i) = dV * v(b, irho, k, j, i);`?

Reply:
No, actually.  But this reminds me that I never removed "rhs" as a variable, which is no longer required.  Thanks.